### DISCUSSION TITLE
about

### DISCUSSION THREAD

**Discussion by hxlog** [hxlog/prologue.dev#37] [Announcements]
作者槐序，00后，在乐观理性与抑郁现实摇摆

**Comment by AngusWG** [comment]
大佬的这个订阅链接 https://prologue.dev/rss 好像不能被订阅，格式看着也很奇怪，应该是xml的？我用的 ttrss订阅，应该和普通订阅器差不多，还是说我的订阅方式有问题？

**Reply by organwalk** [reply]
这个是JSON Feed，不是xml，你可以使用支持JSON Feed的订阅器，或者自己写一个

**Reply by hxlog** [reply]
不好意思，之前看到消息忘记及时调整了 = =

感谢提醒，之前为适配全文输出修改了输出格式，换到JSON Feed。

旧的阅读器一般只支持ATOM和RSS2纯XML格式。

刚改了路由，输出三种不同RSS格式：

[RSS2 Feed](https://prologue.dev/rss)

[ATOM Feed](https://prologue.dev/atomfeed)

[JSON Feed](https://prologue.dev/jsonfeed)

**Comment by QF1007** [comment]
邮箱的链接似乎失效了，尝试发送邮件失败= =

**Comment by Melusinn** [comment]
槐序你好，归档页面的页码块好像有一点bug，第二页之后的页码无法继续翻阅，可以修一下~

**Reply by hxlog** [reply]
不好意思，没及时看到。

这是因为Next.js 15+有个breaking change， params现在是 Promise，必须 await 才能访问。

```
// /src/app/blog/page/[page]/page.js
export default async function PostsPage(context) {
  const { params: { page } } = context
  const pageNumber = parseInt(page) 
```
修改成
```
// /src/app/blog/page/[page]/page.js
export default async function PostsPage({ params }) {
  const { page } = await params
  const pageNumber = parseInt(page) 
```

希望能帮到:)
